.github/workflows/snyk.yml

```diff
@@ -13,8 +13,5 @@ jobs:
       DEBUG: true
       ORG: guardian-devtools
       SKIP_NODE: true
-      
-      
-      JAVA_VERSION: 8
```

Comment:
I'm not 100% sure what this setting means, having been slightly burnt by `snyk monitor` in [May 2021](https://github.com/guardian/ophan/commit/08ab44d454294f4026db7a46cf19a9dda8b9c20b), as that command started a `snyk/snyk:scala` docker image that only had Java **8**, regardless of what Java version was being used for the rest of the build process - but that may be unrelated, or fixed now. I'm willing to give this a go, and you're right, we don't _want_ to be using Java 8!